benchflow-dsl/src/test/scala/cloud/benchflow/dsl/definition/configuration/goal/explorationspace/ExplorationSpaceTest.scala

```diff
@@ -19,6 +19,15 @@ class ExplorationSpaceTest extends JUnitSuite {
       |   values: [1,2,10,32]
     """.stripMargin
 
+  private val servicesExplorationSpaceYaml: String =
```

Comment:
Here is a test for that: https://github.com/benchflow/benchflow/blob/feature-dsl-docker-compose-generation/benchflow-dsl/src/test/scala/cloud/benchflow/dsl/dockercompose/DockerComposeYamlBuilderTest.scala#L15